src/vic3_world/buildings/buildings.h

```diff
@@ -34,6 +35,8 @@ class Buildings
       }
       return {};
    }
+   [[nodiscard]] const std::optional<vic3::Building> GetBuildingInState(const std::string& building_type,
```

Comment:
nit:
reverse parameter order to match parameter of building::GetBuildings(state_id)

Reply:
Done.